addons/website_sale_delivery/views/website_sale_delivery_templates.xml

```diff
@@ -28,21 +28,15 @@
     <template id="payment_delivery_shipping_method" name="Delivery Shipping method" inherit_id="website_sale.address_on_payment">
         <xpath expr="//div[@id='shipping_and_billing']" position="inside">
             <t t-set="delivery_nb" t-value="deliveries and len(deliveries) or 0"/>
-            <div t-if="delivery_nb == 1" id="delivery_carrier" class="mt4">
-                <b>Shipping Method: </b>
-                <t t-foreach="deliveries" t-as="delivery">
-                    <t t-call="website_sale_delivery.payment_delivery_methods"/>
-                </t>
-            </div>
         </xpath>
     </template>
 
     <template id="payment_delivery" name="Delivery Costs" inherit_id="website_sale.payment">
         <xpath expr="//div[@id='payment_method']" position="before">
             <div t-if="deliveries" id="delivery_carrier">
                 <t t-set="delivery_nb" t-value="len(deliveries)"/>
-                <h3 t-if="delivery_nb &gt; 1" class="mb24">Choose a delivery method</h3>
-                <div t-if="delivery_nb &gt; 1" class="card border-0" id="delivery_method">
+                <h3 class="mb24">Choose a delivery method</h3>
```

Comment:
I know the class was already here before you modified this line but can you check`mb24` is real CSS class ?
I see it's used in multiple templates but I don't find any rules for it ?:/